support-frontend/assets/components/orderSummary/contributionsOrderSummary.tsx

```diff
@@ -170,7 +170,7 @@ export function ContributionsOrderSummary({
 	const formattedTotal = simpleFormatAmount(currency, total);
 	const formattedTotalExcludingPromo = simpleFormatAmount(
 		currency,
-		totalExcludingPromo ?? '',
+		totalExcludingPromo ?? 0,
```

Comment:
I'm not 100% sure why this is the case, but this refactor should amount to the same thing.